source/kernel/trace_inode.h

```diff
@@ -0,0 +1,61 @@
+#ifndef TRACE_INODE_H_
+#define TRACE_INODE_H_
+
+#include "trace.h"
+
+/**
+ * @file
+ *
+ * @brief inodes tracer
+ *
+ * It provides tracing of following inodes properties:
+ * - Inode's name; It keeps an internal cache with information if given inode
```

Comment:
We trace:
- Inode's name
- Inode's ID
- Inode's parent ID
- Inode's device ID